client.go

```diff
@@ -433,3 +433,118 @@ func (e GenericOpenAPIError) Body() []byte {
 func (e GenericOpenAPIError) Model() interface{} {
 	return e.model
 }
+
+// Below are the custom Mux API Error types, so that users of the SDK can identify
+// what issue they're running into more easily.
+
+// Generic 400 error
+type BadRequestError struct {
+	body  []byte
+	error string
+}
+
+func (e BadRequestError) Error() string {
```

Comment:
Quick question @GeneticGenesis, can we get access to the Body for the Bad Request error? Or will it always be empty and we rely on the output of the err string?